.github/autobump.txt

```diff
@@ -1474,7 +1474,6 @@ readpe
 rebar3
 recode
 recoverpy
-redis
```

Comment:
Maybe worth updating the livecheck to be pinned on 7.2 given it's going to be supported for quite a while still?

Reply:
Livecheck mismatches also act as a useful tool so that if someone bumps to 7.4 manually then the livecheck CI should pick it up.

Reply:
it might be also better to introduce odie version check rather than using the code comment, I will keep these in mind whenever there is a new release.